experiment/contractNipopow.sol

```diff
@@ -24,6 +24,7 @@ contract Crosschain {
         address payable author;
         uint256 expire;
         Nipopow proof;
```

Comment:
proof is still in storage, though unset

Reply:
This it is gone, but the old code of contesting proof still existed at this point, and it needed Nipopow struct

Reply:
Contesting proof is still at the old version here. It needs the Nipopow structure.